open_contrib_pr.py

```diff
@@ -17,21 +17,34 @@
     pr_title = env_vars.pr_title
     repos_json_location = env_vars.repos_json_location
     token = env_vars.gh_token
+    gh_app_id = env_vars.gh_app_id
+    gh_app_installation_id = env_vars.gh_app_installation_id
+    gh_app_private_key_bytes = env_vars.gh_app_private_key_bytes
+    ghe = env_vars.gh_enterprise_url
+    gh_app_enterprise_only = env_vars.gh_app_enterprise_only
 
     # Auth to GitHub.com
     github_connection = auth.auth_to_github(
-        env_vars.gh_app_id,
-        env_vars.gh_app_installation_id,
-        env_vars.gh_app_private_key_bytes,
-        env_vars.gh_enterprise_url,
         token,
+        gh_app_id,
+        gh_app_installation_id,
+        gh_app_private_key_bytes,
+        ghe,
+        gh_app_enterprise_only,
     )
 
+    if not token and gh_app_id and gh_app_installation_id and gh_app_private_key_bytes:
+        token = auth.get_github_app_installation_token(
+            ghe, gh_app_id, gh_app_private_key_bytes, gh_app_installation_id
+        )
+
+    endpoint = ghe if ghe else "github.com"
```

Comment:
we're sure `ghe` here doesn't include the protocol? aka `https://`

Reply:
Hello @jmeridth,
I'm not 100% sure but I think that I've tested on the evergreen action that if the `GH_ENTERPRISE_URL` variable does not include `https://` the login would fail in the `github3.github.GitHubEnterprise(url=ghe)`function

Reply:
@jmeridth just confirmed running the code, if no Schema is provided on the `GH_ENTERPRISE_URL` the following error happens:
<img width="1532" alt="image" src="https://github.com/user-attachments/assets/43e2f227-136b-467c-811f-74d462a8b76d">

Reply:
My bad, I was not seeing what was happening, just updated the code to strip the `https://` from ghe otherwise the following commands will not work